### ISSUE TITLE
BUG:  When monitorring NGINX on windows, UI become unresponsive

### ISSUE THREAD

**Issue by seanvree** [Monitorr/Monitorr#116]
## Your environment:

- **Operating system**: Windows 10

- **Webserver**:       NGIX v. 3.8

- **PHP Version**:      7.2.1 x64 NTS

- **Monitorr Branch**:   Develop

## Describe your issue:

From DISCORD conv w/ @Nomelas 

**ISSUE** : When monitrring NGINX on windows (Port 80) UI becomes unresponsive.  

**NOTE** :  there's actually NO reason to Monitorr you own webserver because if webserver becomens unresponsive or DOWN,  monitorr wouldn't work anyway. That's not the proper answer, move of just a FYI. 

Logs:

```
PHP Warning:  curl_close(): supplied resource is not a valid cURL handle resource in C:\nginx\html\Monitorr\assets\php\check.php on line 155
PHP Warning:  curl_close(): supplied resource is not a valid cURL handle resource in C:\nginx\html\Monitorr\assets\php\check.php on line 155
```
```
2018/02/02 21:02:56 [error] 48376#45144: *28 FastCGI sent in stderr: "PHP Notice:  A non well formed numeric value encountered in C:\nginx\html\Monitorr\assets\php\functions.php on line 219
PHP Notice:  A non well formed numeric value encountered in C:\nginx\html\Monitorr\assets\php\functions.php on line 221
PHP Notice:  A non well formed numeric value encountered in C:\nginx\html\Monitorr\assets\php\functions.php on line 223
PHP Notice:  A non well formed numeric value encountered in C:\nginx\html\Monitorr\assets\php\functions.php on line 225" while reading response header from upstream, client: 192.168.2.20, server: jefflix.stream, request: "GET /monitorr/assets/php/systembadges.php HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.2.10", referrer: "https://192.168.2.10/monitorr/"
```

**Comment by seanvree** [comment]
May be related to https://github.com/Monitorr/Monitorr/issues/111  as they are BOTH windows and NGINX.

**Comment by seanvree** [comment]
@jonfinley  can you test this on your box?  I know you're running LINUX, but I wanna see if this happens to you.  Just add your NGINX port to your config.php

**Comment by seanvree** [comment]
fixed with version 1.5.0m

https://github.com/Monitorr/Monitorr/releases
